.github/workflows/ci.yml

```diff
@@ -5,35 +5,28 @@ on:
 - pull_request
 
 jobs:
+  ruby-versions:
+    uses: ruby/actions/.github/workflows/ruby_versions.yml@master
+    with:
+      min_version: 2.4
+
   host:
-    name: ${{ matrix.os }} ${{ matrix.ruby }}
+    needs: ruby-versions
+    name: build (${{ matrix.ruby }} / ${{ matrix.os }})
```

Comment:
Can we keep the original `name:` for short label?

Reply:
Yes, now done!